cmake/modules/AddSwift.cmake

```diff
@@ -159,7 +159,7 @@ function(_add_variant_link_flags
       result)
 
   if("${sdk}" STREQUAL "LINUX")
-    list(APPEND result "-lpthread" "-ldl")
+    list(APPEND result "-lpthread" "-ldl" "-Wl,-Bsymbolic")
```

Comment:
Yes, it is.  After 2.2 is forked, I'm going to work with @tienex on a different (probably better) solution based upon the gold linker.  I'll restrict this to arm-linux.